### PR TITLE
release Flutter SDK 1.1.0

### PR DESCRIPTION
**Pull request by GhadaDataDome**
This version includes:
* Migrate to Android V2
* upgrade Dart SDK version `sdk: '>=2.12.0 <4.0.0'` to be compatible to Dart 3
* Upgrade kotlin version to 1.6.10
* Upgrade CompileSdkVersion and targetSdkVersion to 32
* Upgrade DataDome SDK Core versions: 
    -  Change Android native SDK dependency from jitpack to maven
    - Import Android native SDK 1.8.0
    - Upgrade iOS SDK version to 3.2.2 (with pod upgrade DataDomeSDK)
 Those versions embed several improvements including Event Tracking throttling
* Upgrade gradle to version 7.1.2
* Upgrade okhttp library to 4.9.0 
* Define android:exported value in AndroidManifest.xml file
* Replace internal testing url and client key by a static text ('test_url')

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+## 1.1.0
+
+* Migrate to Android V2
+* upgrade Flutter SDK version for versions < 4.0.0
```
Comment (darahak-datadome) [inline CHANGELOG.md]:
```suggestion
* Upgrade Flutter SDK version for versions < 4.0.0
```

Comment (darahak-datadome) [reply CHANGELOG.md]: > for versions < 4.0.0

Can you elaborate on which component you are talking about here?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+* Upgrade kotlin version to 1.6.10
```
Comment (darahak-datadome) [inline CHANGELOG.md]:
```suggestion
* Upgrade Kotlin version to 1.6.10
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,11 @@
...
+* Upgrade CompileSdkVersion and targetSdkVersion to 32
```
Comment (darahak-datadome) [inline CHANGELOG.md]:
```suggestion
* Upgrade `compileSdkVersion` and `targetSdkVersion` to 32
```

File: CHANGELOG.md (hunk 1 above)
Comment (GhadaDataDome) [reply CHANGELOG.md]: To be compatible with Dart 3 features, we upgrade the Dart SDK version with `sdk: '>=2.12.0 <4.0.0'`

**Review by darahak-datadome — APPROVED** [review]

**Pull request merged**